### DISCUSSION TITLE
Is there a plan to add support for Software Packages?

### DISCUSSION THREAD

**Discussion by inventron-tr** [aws/aws-iot-device-sdk-cpp-v2#739] [Q&A] [answered]
Is there a plan to add support for Software Packages?
$aws/things/DeviceName/shadow/name/$package can be managed like the shadow implementations

**Comment by bretambrose** [comment, marked as answer]
It keeps dropping below the cutoff line priority-wise (internally).  If this is important to you, I would suggest engaging an AWS TAM or solutions partner with information about your use case to help potentially re-prioritize.
